src/main/java/org/maxgamer/rs/structure/ReflectUtil.java

```diff
@@ -4,72 +4,62 @@
 import java.lang.reflect.Modifier;
 import java.util.Arrays;
 
-public class ReflectUtil{
-	public static String describe(Object o){
-		StringBuilder result = new StringBuilder();
-		String newLine = System.getProperty("line.separator");
+public class ReflectUtil {
+    public static String describe(Object o) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "describe" is 15 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUndocumentedApi)